File: LeetCode/Easy/0027-remove-element/0027-remove-element.go
func removeElement(nums []int, val int) int {
    n := len(nums)
    cnt := 0
    for i := 0; i < n; i++ {
        if (nums[i] == val) {
            nums[i] = 51
            cnt += 1 
        }
    }
    sort.Ints(nums)
    return n - cnt
}

File: LeetCode/Easy/0027-remove-element/0027-remove-element.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int n = nums.length;
        int k = n;
        int REMOVED_FLAG = 51;
        for (int i = 0; i < n; i++) {
            if (nums[i] == val) {
                k--;
                nums[i] = REMOVED_FLAG;
            }
        }
        Arrays.sort(nums);
        return k;
}   
}